src/pages/CoursePage/CoursePage.tsx

```diff
@@ -48,7 +48,7 @@ const CoursePage = () => {
                 variant="contained"
                 color="primary"
                 onClick={() => {
-                  navigate('topic/' + topic.id)
+                  navigate('Topic/' + topic.id)
```

Comment:
in der url sollte alles lowercas sein
```suggestion
                  navigate('topic/' + topic.id)
```